subsys/bluetooth/host/classic/l2cap_br.c

```diff
@@ -238,18 +241,69 @@ static uint8_t l2cap_br_get_ident(void)
 	return ident;
 }
 
+static void raise_data_ready(struct bt_l2cap_br_chan *br_chan)
+{
+	if (!atomic_set(&br_chan->_pdu_ready_lock, 1)) {
+		sys_slist_append(&br_chan->chan.conn->l2cap_data_ready,
+				 &br_chan->_pdu_ready);
+		LOG_DBG("data ready raised");
+	} else {
+		LOG_DBG("data ready already");
+	}
+
+	bt_conn_data_ready(br_chan->chan.conn);
+}
+
+static void lower_data_ready(struct bt_l2cap_br_chan *br_chan)
+{
+	struct bt_conn *conn = br_chan->chan.conn;
+	__maybe_unused sys_snode_t *s = sys_slist_get(&conn->l2cap_data_ready);
```

Comment:
Was not aware of `__maybe_unused` - Sounds like a nice alternative to `(void)s`

Reply:
I wonder if Coverity will still complain - we'll see :)

Reply:
some helpful person suggested it here https://github.com/zephyrproject-rtos/zephyr/pull/72854#issuecomment-2133684635